src/NUnitConsole/nunit3-console.tests/CommandLineTests.cs

```diff
@@ -522,6 +523,18 @@ public void ShouldSetTeamCityFlagAccordingToArgsAndDefauls(bool hasTeamcityInCmd
 
         #endregion
 
+        #region Testlist Option
+
+        [Test]
+        public void ShouldNotFailOnEmptyLine()
+        {
+            var testListPath = Path.Combine(TestContext.CurrentContext.TestDirectory, "TestListWithEmptyLine.tst");
+            var options = new ConsoleOptions("--testlist=" + testListPath);
+            Assert.That(options.errorMessages, Is.Empty);
+        }
```

Comment:
Good that you added a test. Why not go all the way and verify that the content of TestList?